Python/Bulk_File_Renamer/utils.py

```diff
@@ -0,0 +1,232 @@
+
+
+    def __init__(self, files):
+        """Organizes files object in a particular order."""
+        if files is None or len(files) == 0:
+            print("0 files to organize. Exiting")
+            exit(0)
+        self.files = files
+
+    @staticmethod
+    def set_arguments(*names):
+        """Takes input from the user for all arguments in "names"\
+             and returns it in a dictionary"""
+        args = {}
+        for name in names:
+            value = ""
+            while value == "":
+                value = input(f"Enter the value for '{name}' argument: ")
+                if value == "":
+                    print("Error: No value received")
+            args[name] = value
+
+        return args
+
+    @staticmethod
+    def organize_files(files, arguments):
+        """Organizes files based on the arguments passed. Different values of arguments\
+            can be: rename, rename and copy to new folder, delete files"""
+        if arguments["method"] == "rename":
+            count = 1
+            new_name = arguments["value"]["filename"]
+            for file in files.keys():
+                extension = splitext(files[file]["name"])[1]
+                name = f"{new_name}_{count}{extension}"
+                prefix = split(files[file]["path"])[0]
+                count += 1
+
+                rename(files[file]["path"], join(prefix, name))
+
+            print(f"Renamed {len(files)} files successfully!!")
+
+        elif arguments["method"] == "rename-copy":
+            count = 1
+            new_name = arguments["value"]["filename"]
+
+            # Create directory
+            folder_name = arguments["value"]["folder-name"]
+            if not isdir(folder_name):
+                mkdir(folder_name)
+
+            for file in files.keys():
+                extension = splitext(files[file]["name"])[1]
+                name = f"{count}_{new_name}{extension}"
+                prefix_file = split(files[file]["path"])[0]
+                count += 1
+
+                copy2(files[file]["path"], join(folder_name, name))
+
+            print(f"Copied {len(files)} files successfully!!")
+
+        elif arguments["method"] == "delete":
+            confirm = "n"
+            confirm = input(f"Deleting {len(files)}, press Y to confirm: ")
+            if confirm.lower() != "y":
+                print("Exiting")
+                exit(0)
+            else:
+                for file in files.keys():
+                    remove(files[file]["path"])
+
+                print(f"Deleted {len(files)} files successfully!")
+
+    def organize(self):
+        """Driver function to call organize_files method and set relevant arguments."""
+        print(f"\nFiles available to organize: {len(self.files)}")
+
+        arguments = {}
+        choice = input("Choose the method to re-organize:\n1. Rename in-place\n\
+            2. Rename and move to new folder\n3. Delete Files\n: ")
+        if choice == "1":
```

Comment:
You can replace this with switch case and put a default case for invalid choice.

Reply:
Hey. Actually switch case is not a construct in Python, not that I know of. That is why I am using If-Else ladder here

Reply:
Sure, I am working on the README file next